README.md

```diff
@@ -75,51 +75,32 @@ Building source tarball and linux packages for new releases
 -----------------------------------------------------------
 
  - tag the release on the `hhvm-staging` repo
- - `bin/make-all-packages-on-aws VERSION`
+ - `bin/build-on-aws VERSION`
 
 If you just need to rebuild for one distribution, with no code changes:
 
- - edit `DISTRO/PKGVER` if needed (e.g. if a previouis package was pushed for this HHVM version and distribution) and push to github
- - `bin/make-package-on-aws VERSION DISTRO`
```

Comment:
is the azure script still meant to be used directly for mac?

Reply:
I guess the azure script is not needed anymore, since the builds can be triggered through AWS (`bin/build-on-aws BuildAndPublishMacOS $VERSION` works), but running the AWS state machine just to build a Mac binary has a lot of overhead so it's probably still useful to keep the azure script too.